From 88f847b370ae0bab80d415f2ae50d15d6b9f7a1f Mon Sep 17 00:00:00 2001
From: Daniel Wagner <dwagner@suse.de>
Date: Fri, 15 May 2020 13:36:20 +0200
Subject: [PATCH] Revert "sched/rt: Provide migrate_disable/enable() inlines"
Patch-mainline: Never, -RT specific
References: SLE Realtime Extension

Remove the migrate_{enable|disable}() implementation from mainline in
order to allow to add the full featured version from the RT patch
series.

Signed-off-by: Daniel Wagner <dwagner@suse.de>
---
 include/linux/preempt.h | 30 ------------------------------
 1 file changed, 30 deletions(-)

diff --git a/include/linux/preempt.h b/include/linux/preempt.h
index bc3f1aecaa19..bbb68dba37cc 100644
--- a/include/linux/preempt.h
+++ b/include/linux/preempt.h
@@ -322,34 +322,4 @@ static inline void preempt_notifier_init(struct preempt_notifier *notifier,
 
 #endif
 
-/**
- * migrate_disable - Prevent migration of the current task
- *
- * Maps to preempt_disable() which also disables preemption. Use
- * migrate_disable() to annotate that the intent is to prevent migration,
- * but not necessarily preemption.
- *
- * Can be invoked nested like preempt_disable() and needs the corresponding
- * number of migrate_enable() invocations.
- */
-static __always_inline void migrate_disable(void)
-{
-	preempt_disable();
-}
-
-/**
- * migrate_enable - Allow migration of the current task
- *
- * Counterpart to migrate_disable().
- *
- * As migrate_disable() can be invoked nested, only the outermost invocation
- * reenables migration.
- *
- * Currently mapped to preempt_enable().
- */
-static __always_inline void migrate_enable(void)
-{
-	preempt_enable();
-}
-
 #endif /* __LINUX_PREEMPT_H */
-- 
2.26.2

